tests/helpers/start-app.js

```diff
@@ -1,7 +1,7 @@
-import { run } from '@ember/runloop';
-import { merge } from '@ember/polyfills';
 import Application from '../../app';
 import config from '../../config/environment';
+import { merge } from '@ember/polyfills';
+import { run } from '@ember/runloop';
```

Comment:
why did you change the order here?

Reply:
It's also the default output by `ember init`